README.md

```diff
@@ -553,6 +553,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 ## DevOps
  * [ansible-crystal](https://github.com/CorbanR/ansible-crystal) - Ansible playbook for installing crystal
  * [crystal-cookbook](https://github.com/vjdhama/crystal-cookbook) - Chef cookbook for installing crystal
+ * [capistrano-kemal](https://github.com/bitfex/capistrano-kemal) - Capistrano Plugin for Kemal
```

Comment:
What's the problem with the original https://github.com/sdogruyol/capistrano-kemal?

Reply:
Oh, it have many disadvantages:

1. Doesn't support custom filename / binary name
2. Doesn't correctly start app first time
3. Doesn't respect ENV variables
4. Maybe some other bugs

Reply:
And also seems that it's abadoned - it has MR with some important changes that doesn't accepted for a long time